src/core/config/resolvers/paths.ts

```diff
@@ -19,12 +19,28 @@ export async function resolvePathOptions(options: NitroOptions) {
   // Add aliases
   options.alias = {
     ...options.alias,
-    "~/": join(options.srcDir, "/"),
-    "@/": join(options.srcDir, "/"),
-    "~~/": join(options.rootDir, "/"),
-    "@@/": join(options.rootDir, "/"),
+    "~": join(options.srcDir, "/"),
+    "@": join(options.srcDir, "/"),
```

Comment:
Aliases without trailing `/`, specially `@` are not added because it can conflict with npm package names `@foo/bar` and it is unsafe